bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/ThingUID.java

```diff
@@ -168,6 +169,20 @@ public ThingTypeUID getThingTypeUID() {
     }
 
     /**
+     * Returns the thing id.
+     *
+     * @return thing id
+     */
+    public String getThingId() {
```

Comment:
But I think the getId() method can return the bridge(s) - it returns all but the last segment, rather than what this is doing which is to return the first segments...

Maybe there's a better name for the method...